ledger/src/blockstore_processor.rs

```diff
@@ -1135,11 +1135,20 @@ pub struct BatchExecutionTiming {
 
     /// Wall clock time used by the transaction execution part of pipeline.
     /// [`ConfirmationTiming::replay_elapsed`] includes this time.  In microseconds.
-    pub wall_clock_us: u64,
+    wall_clock_us: u64,
 
     /// Time used to execute transactions, via `execute_batch()`, in the thread that consumed the
-    /// most time.
-    pub slowest_thread: ThreadExecuteTimings,
+    /// most time (in terms of total_thread_us) among rayon threads. Note that the slowest thread
+    /// is determined each time a given group of batches is newly processed. So, this is a coarse
+    /// approximation of wall-time single-threaded linearized metrics, discarding all metrics other
+    /// than the arbitrary set of batches mixed with various transactions, which replayed slowest
+    /// as a whole for each rayon processing session, also after blockstore_processor's rebatching.
+    ///
+    /// When unified scheduler is enabled, this field isn't maintained, because it's not batched at
+    /// all and thus execution is fairly evenly distributed across its worker threads in the
+    /// granularity of individual transactions, meaning single-threaded metrics can reliably
+    /// be derived from dividing replay-slot-stats by the number of threads.
```

Comment:
I think it's fine to not implement this metric for new scheduler, but not sure I buy this justification. No matter how well balanced a scheduler is there's always a slowest path.

trivial example: 2 txs, 1 takes 40ms to execute, 1 takes 1us to execute. There's a slowest path here, and the single-threaded metrics cannot be derived from the replay-slot-stats

Reply:
Agree generally this metric is not applicable due to lack of batching, but saying the execution is fairly balanced and you can derive the single-threaded stats is misleading imo.

Just state that this metric is tracking slowest batch-to-batch time, which doesn't apply if we're not batching.

Reply:
good point. I removed the controversial part: b61c19302198fae15f4ad4e92a6c95a254479acf